setup.cfg

```diff
@@ -14,3 +14,13 @@ max-line-length = 80
 max-line-length = 80
 ignore = F401,E402,F403,F811
 exclude = venv
+
+[report]
+exclude_lines =
+    pragma: no cover
+    def __repr__
```

Comment:
Not sure it makes effect

Reply:
I copy-pasted the thing from the docs :)
https://coverage.readthedocs.io/en/coverage-4.2/excluding.html

but yes, it does not work somehow.

Reply:
Or just add a quick test with default params